extensions/interactions/FractionInput/directives/FractionInputValidationService.js

```diff
@@ -17,16 +17,28 @@
  */
 
 oppia.factory('FractionInputValidationService', [
-  'WARNING_TYPES', 'baseInteractionValidationService', 'FractionObjectFactory',
+  'WARNING_TYPES', 'baseInteractionValidationService',
+  'FractionObjectFactory',
   function(
     WARNING_TYPES, baseInteractionValidationService, FractionObjectFactory) {
+    getNonIntegerInputWarning = function(i, j) {
+      return {
+        type: WARNING_TYPES.ERROR,
+        message: (
+          'Rule ' + (j + 1) + ' from answer group ' +
+          (i + 1) + ' is invalid as input must be an ' +
```

Comment:
... is invalid: the input value should be an integer.

Reply:
Done.